main.yml

```diff
@@ -4,6 +4,23 @@
 # Note, 'local' (the host name in the inventory)
 #   should be assigned ``ansible_connection=local``.
 
+# Ansible requires python2, while Ubuntu >=15 strictly uses Python3.
```

Comment:
I installed against a fresh install of 16.04, and did no need this. python is still 2.7 on 16.04.